tracer/src/Datadog.Trace/Util/HexString.cs

```diff
@@ -16,6 +16,9 @@
 using BitConverter = Datadog.Trace.Util.BitConverterShim;
 using MemoryMarshal = Datadog.Trace.VendoredMicrosoftCode.System.Runtime.InteropServices.MemoryMarshal;
 #endif
+#if NETFRAMEWORK || NETSTANDARD2_0
+using Unsafe = Datadog.Trace.VendoredMicrosoftCode.System.Runtime.CompilerServices.Unsafe.Unsafe;
+#endif
```

Comment:
Isn't this going to cause issues when we go to `netcoreapp2.1` in v3 because of the extra dll issue?

```suggestion
#if !NETCOREAPP3_0_OR_GREATER
using Unsafe = Datadog.Trace.VendoredMicrosoftCode.System.Runtime.CompilerServices.Unsafe.Unsafe;
#endif
```